setup.py

```diff
@@ -394,8 +404,6 @@ def initialize_options(self):
     def finalize_options(self):
         """Post-process options."""
         if len(self.md_doc_paths) == 0:
-            print("no paths were specified, using default markdown file paths for pyspec"
```

Comment:
does this PR change anything if someone does provide non-default paths?

Reply:
I'm not sure. I don't even know how to provide a non-default path. Is this something we do?

Reply:
not sure if its used but it does seem like the code currently supports it

Reply:
Ah, so it doesn't change any functionality with that. I just removed the print call.

Reply:
I haven't used the path flags, but Proto did when he designed it: https://github.com/ethereum/consensus-specs/tree/v1.5.0-alpha.6/tests/core/pyspec#advanced

Given that no current use cases require us to *only* build phase0 specs, I think it’s OK to ignore it for now.